README.md

```diff
@@ -103,7 +120,7 @@ You can also install a specific version of the collection, for example, if you n
 ansible-galaxy collection install NAMESPACE.COLLECTION_NAME:==0.1.0
 ```
 
-See [Ansible Using collections](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html) for more details.
+See [Ansible using collections](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html) for more details.
```

Comment:
"Ansible using collections" sounds like a statement (like "Ansible is now using collections").

How about this:
```suggestion
See [Ansible's Using collections](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html) for more details.
```
? Or
```suggestion
See ["Using collections" in the Ansible documentation](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html) for more details.
```
?

And now that I actually looked at this link, how about simply using it's title, since it contains all three words, but in a different order:
```suggestion
See [using Ansible collections](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html) for more details.
```
?

Reply:
i like the latter, committed thanks!